packages/composite-checkout-wpcom/src/components/form-field-annotation.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import styled from '@emotion/styled';
+
+type FormFieldAnnotationProps = {
+	// Semantic props
+	labelText: string;
+	descriptionText?: string;
+	errorMessage: string;
+
+	// Functional props
+	isError: boolean;
+	isDisabled: boolean;
+
+	// Technical props
+	className?: string;
+	labelId: string;
+	descriptionId: string;
+	formFieldId: string;
+};
+
+export const FormFieldAnnotation: FunctionComponent< FormFieldAnnotationProps > = ( {
```

Comment:
We usually export React components as default exports -- especially in a module like this that doesn't export anything else.

Libraries for code-splitting of React components often make use of that convention:
```js
const FormFieldAnnotation = React.lazy( () => import( './form-field-annotation' ) );
```